Sources/MintKit/Mint.swift

```diff
@@ -279,9 +281,14 @@ public class Mint {
 
     private func buildPackage(name: String, path: Path) throws {
 
-        let osVersion = ProcessInfo.processInfo.operatingSystemVersion
-        let target = "x86_64-apple-macosx\(osVersion.majorVersion).\(osVersion.minorVersion)"
-        let command = "swift build -c release -Xswiftc -target -Xswiftc \(target) -Xswiftc -static-stdlib"
+        #if os(macOS)
+            let target: String
```

Comment:
Is there a reason why this isn't just `let target = ` on line 287?

Reply:
Not, not really. This is a holdover from me trying to extract some common parts, but ended up just having 2 completely seperate paths. Good catch